lib/block_view.go

```diff
@@ -117,6 +118,11 @@ type UtxoView struct {
 
 	// Validator mappings
 	ValidatorPKIDToValidatorEntry map[PKID]*ValidatorEntry
+	// Note: I really don't like using a string for the map key here,
+	// as it loses the intent. However, bls.PublicKey does not implement
+	// the comparable interface for golang maps and implementing it
+	// would require greater changes to the BLS package, which isn't worth it.
+	ValidatorVotingPublicKeyToBLSPublicKeyPKIDPairEntry map[bls.SerializedPublicKey]*BLSPublicKeyPKIDPairEntry
```

Comment:
Same comment as in db_utils. You can name this ValidatorBLSPublicKeyPKIDPairEntry. No need for the VotingPublicKey part since that's not mentioned anywhere else. IMO it's preferable to be concise and clear than to have the name be the most technically correct
```suggestion
	ValidatorBLSPublicKeyPKIDPairEntry map[bls.SerializedPublicKey]*BLSPublicKeyPKIDPairEntry
```